scripts/install.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+# Copyright 2024 Rapyuta Robotics
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Function to print usage
+### Prints the usage instructions and exits
+usage() {
+  echo -e "\033[1;31mUsage: $0 [<release tag>]\033[0m"
+  echo -e "\033[1;31mIf no release tag is specified, the latest release will be used.\033[0m"
+  echo -e "\033[1;31mExample: $0 v1.0.0\033[0m"
+  exit 1
+}
+
+# Check if more than one argument is provided
+### Ensure only one argument is provided, otherwise print usage and exit
+if [ "$#" -gt 1 ]; then
+  usage
+fi
+
+# Set the repository base URL (hardcoded)
```

Comment:
suggestion: Consider printing the rapyuta.io trademark banner that we added in the onboarding script.